x-pack/plugins/cases/public/components/case_view/case_view_page.tsx

```diff
@@ -411,9 +403,6 @@ export const CaseViewPage = React.memo<CaseViewPageProps>(
                   connectorName={connectorName}
                   connectors={connectors}
                   hasDataToPush={hasDataToPush && userCanCrud}
-                  hideConnectorServiceNowSir={
```

Comment:
@cnasikas I don't think we need this prop anymore because I believe we hid the SIR connector to avoid having to send over a bunch of alerts attached to a sub case.

Reply:
Correct!